package.json

```diff
@@ -20,53 +20,53 @@
   "license": "MIT",
   "dependencies": {
     "@types/mongodb": "^3.5.27",
-    "bson": "^1.1.4",
+    "bson": "^4.4.0",
```

Comment:
We can't merge this, because that would cause conflicts between the mongodb package's version of bson and Mongoose's.